e2e_tests/tests/experiment/test_noop.py

```diff
@@ -252,36 +252,42 @@ def test_startup_hook() -> None:
     )
 
 
-@pytest.mark.e2e_cpu  # type: ignore
-def test_rng_restore() -> None:
+def _test_rng_restore(fixture: str, metrics: list) -> None:
+    """
+    This test confirms that an experiment can be restarted from a checkpoint
+    with the same RNG state. It requires a test fixture that will emit
+    random numbers from all of the RNGs used in the relevant framework as
+    metrics. The experiment must have a const.yaml, run for at least 3 steps,
```

Comment:
q: does this use validation metrics?

Reply:
No, I think they're all training metrics